yt/yt/server/node/exec_node/job.cpp

```diff
@@ -2394,6 +2394,11 @@ TJobProxyInternalConfigPtr TJob::CreateConfig()
         }
     }
 
+    if (UserJobSpec_) {
+        proxyConfig->ContainerMemoryLimit = UserJobSpec_->container_memory_limit();
+        proxyConfig->JobProxyMemoryLimit = UserJobSpec_->job_proxy_memory_limit();
```

Comment:
Job proxy memory limit is useful even if there is no user job

Reply:
If we limit them both together then job proxy without user job could be limited too.